docs/Components/SudokuBoard/BoardVariants.md

```diff
@@ -4,6 +4,13 @@
 
 Sudoku game with 9x9 grid.
 
+Below are the difficulties for Classic variant, from easiest to most challenging:
+
+- novice
+- trainee
+- amateur
+- layman
+
```

Comment:
This isn't a bug with Frontend. And we aren't sure there is a bug at this point in time.

Reply:
I can verify having manually done a layman puzzle and inspected a novice puzzle that they are indeed switched. While the source of the bug is indeed upstream I would still argue it is a bug to have the puzzles mislabeled even if the labeling process isn't owned by the frontend. i.e. it is a bug from a user POV ignoring the technical details. Since it is going on main I would still advocate for a comment documenting it.

Reply:
A todo comment would be nice since one it's fixed upstream the fixes will have to be applied to the frontend

Reply:
I guess it wouldn't hurt to document it, I'll push something up.

Reply:
It is pushed up.